ansible/roles_ocp_workloads/ocp4_workload_le_certs_cert_manager/tasks/workload.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Install cert-manager chart
+  command: "{{ item }}"
+  loop:
+    - "helm repo add jetstack https://charts.jetstack.io"
+    - "helm repo update"
+    - "helm install \
+          cert-manager jetstack/cert-manager \
+          --namespace {{ cert_manager_ns }} \
+          --create-namespace \
+          --version {{ cert_manager_helm_ver }} \
+          --set installCRDs=true \
+          --set 'extraArgs={--dns01-recursive-nameservers=8.8.8.8:53\,1.1.1.1:53,--dns01-recursive-nameservers-only}'"
+  register: cert_manager_helm_install_result
+  until: cert_manager_helm_install_result.rc == 0
+  ignore_errors: true
+  retries: 3
+  delay: 10
+
+- name: Install cert-manager-configs  chart
+  command: "{{ item }}"
+  loop:
+    - "helm repo add redhat-cop https://redhat-cop.github.io/helm-charts"
```

Comment:
Why? There are official helm charts. Why use cop ones?

Reply:
that one has a prepackaged configuration for cert-manager (ClusterIssuer, Certficates etc.). Idea was to wrap it all in single Chart instead of keeping templates files in the role itself